docs/release-notes/index.md

```diff
@@ -20,318 +21,609 @@ To check for security updates, go to [Security announcements for the Elastic sta
 % ### Fixes [elasticsearch-next-fixes]
 % *
 
-## 9.1.0 [elasticsearch-910-release-notes]
+## 9.0.1 [elasticsearch-901-release-notes]
```

Comment:
@brianseeders remember per https://github.com/elastic/docs-content/issues/1152 we need these to be in `X.Y.Z` format going forward so these IDs should be `9.0.1`